providers/yttcc/lib/helpers.star

```diff
@@ -323,15 +323,25 @@ end
 
 # get_labels_map_from_string constructs a map from given string of the format "key1=label1,key2=label2"
 def get_labels_map_from_string(labelString):
-   labelMap = {}
-   for val in labelString.split(','):
+   return get_map_from_string(labelString, ',')
+end
+
+# get_extra_args_map_from_string constructs a map from given string of the format "key1=label1,key2=label2"
+def get_extra_args_map_from_string(labelString):
+   return get_map_from_string(labelString, ';')
```

Comment:
Why not use ',' as the delimiter?

Reply:
As you can see, ',' is already used as the delimiter of some flags, e.g. cipher-suites `TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305,TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305,TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384`